webapp/channels/src/components/admin_console/admin_definition.jsx

```diff
@@ -3100,6 +3100,91 @@ const AdminDefinition = {
                 ],
             },
         },
+        wrangler: {
+            url: 'site_config/wrangler',
+            title: t('admin.sidebar.wrangler'),
+            title_default: 'Wrangler',
+            isHidden: it.any(it.not(it.userHasReadPermissionOnResource(RESOURCE_KEYS.SITE.POSTS)), it.configIsFalse('FeatureFlags', 'MoveThreadsEnabled')),
+            schema: {
+                id: 'WranglerSettings',
+                name: t('admin.site.wrangler'),
+                name_default: 'Wrangler',
+                settings: [
+                    {
+                        type: Constants.SettingsTypes.TYPE_ROLES,
+                        multiple: true,
+                        key: 'WranglerSettings.PermittedWranglerUsers',
```

Comment:
The name of the config is misleading (for the developer). I would expect this to be a list of usernames, not a list of user roles.

Reply:
Changed to `PermittedWranglerRoles`